test/project1/source/colors/literals/color-alpha-mixed-values.brs

```diff
@@ -0,0 +1,7 @@
+sub init()
+    color = "0xf00000"
```

Comment:
Tests should include non-colour strings matching our logic - short / long strings including colour-like information, things which could be colours but aren't,...

Reply:
Done. Let me know if we need some extra examples